### ISSUE TITLE
Cleaner Cockpit

### ISSUE THREAD

**Issue by glowf1sh** [flybywiresim/aircraft#5610]
### Aircraft Version

Development

### Description

Hello,

as of dev d30667b and the last two releases, the cockpit looks pretty dirty and used.

I know its general personal taste and some people think it has to do something with immersion if you have all these stains and dirty areas. But to me it is disgusting and ruining my "fun" in the simulator. I am generally a person who keeps his car and everything clean and tidy and i dont see the reason, why a multi-million dollar machine should look like a trashbin.

Maybe you can make the "used/dirty" look as an option for those who want it. Otherwise i´d prefer the old and "normal" look. The old versions had a cockpit with fingerprints and scratches on the windows too but it was ok as it was realistic. But now you are exaggerate a little in my humble opinion.

(I couldnt attach screenshots for some reason - Github says "Something went really wrong, and we can´t process that file" for a regular jpg or png). To be more specific of what i´m talking about: I think the following sections are too dirty or the worn down look is too extreme:

- Overhead Panel: Strange white foggy/cloudy stains around all buttons like GEN1, APU GEN, BUS TIE, EXT PWR etc. Also the BAT 1 and BAT 2 Voltage Displays are surrounded by these white cloudy stains that make no sense to me. Why should the painted metal instrument cover look like this on the corners and outside of all buttons? Maybe if someone would use some really agressive brake cleaner to clean these instruments. But its not realistic. Remember .. a plane is a very expensive and very well maintained item. No sane person would treat it in a way that is ruining the look. It would even be against all operating and maintainance manuals to clean the area with agressive chemicals. So these stains make no sense.

- ENG Instrument Panel for starting ENG 1 and ENG 2 below the throttle: Same strange stains and also some blue writings on the panel that seem to be a graphical mistake. Under the Word "ENG" there is a small blue + Symbol and right next to the "MASTER 2 ON" writing there is some strange blue writing i cannot read. It probably says "115 VU" or something. It surely is an error in the texture.

- Foot / Pedal Section: The metal cover plates on the floor where the pilots an rest their feet and put their shoes most of the time look very stained and dirty. Might be realistic - please have someone clean it. I dont want to fly in such a dirty plane ^^

- My negative favorite - PF side (left), next to the Window and the PFD OFF/BRT, LOUD SPKR OFF/MAX, PFD/ND XFR, CONSOLE/FLOOR BRT/DIM/OFF Knobs, you have a hole in the frame that is obviously a Pen Holder. Pilot can put a pen inside that hole. You painted way too much traces of ballpoint paint around that hole. It looks horrible.

Sorry for the critical/negative Feedback. I really enjoy the Mod and its awesome and i have alot of fun because of your guys great work. Thank you so much!

**Comment by sidnov** [comment]
An average 320neo makes nearly 5-6 trips each day. It is on ground barely for a few hours each day, making it impossible for pilots, ground crew, engineers etc. to keep their cockpits clean and tidy. Thus, we decided to implement the wear in the textures to match the realism

Your feedback and suggestion to add an option to switch between textures is valid, but it would not be a priority at this point. I would leave the issue open either way, for someone to come back and take a look later.

**Comment by sidnov** [comment]
>ENG Instrument Panel for starting ENG 1 and ENG 2 below the throttle: Same strange stains and also some blue writings on the panel that seem to be a graphical mistake. Under the Word "ENG" there is a small blue + Symbol and right next to the "MASTER 2 ON" writing there is some strange blue writing i cannot read. It probably says "115 VU" or something. It surely is an error in the texture.

FYI, this is not an error

**Comment by glowfishDE** [comment]
> > ENG Instrument Panel for starting ENG 1 and ENG 2 below the throttle: Same strange stains and also some blue writings on the panel that seem to be a graphical mistake. Under the Word "ENG" there is a small blue + Symbol and right next to the "MASTER 2 ON" writing there is some strange blue writing i cannot read. It probably says "115 VU" or something. It surely is an error in the texture.
> 
> FYI, this is not an error

You are right ! I just checked. There is some blue writing in the real world too. But unfortunately the images are too bad. What does it say? And shouldnt it be more readeable in the virtual airbus cockpit ? It looks like the font is a little distorted.

**Comment by glowfishDE** [comment]
> An average 320neo makes nearly 5-6 trips each day. It is on ground barely for a few hours each day, making it impossible for pilots, ground crew, engineers etc. to keep their cockpits clean and tidy. Thus, we decided to implement the wear in the textures to match the realism
> 
> Your feedback and suggestion to add an option to switch between textures is valid, but it would not be a priority at this point. I would leave the issue open either way, for someone to come back and take a look later.

Thanks @sidnov - yeah i know.. its always a personal thing.. some like it more used, others like it more new.. so in the end you cannot make everyone happy about it (unless not having such an option to change the look - which is probably a catastrophe to maintain and code).. So i see your point and i was expecting that answer. However i wanted to express my thoughts as i´m regulary flying (daily) with friends after work and we all agree that the new worn look is something we dont like so much (sorry).

Thanks for your great work though and have a nice evening for now :)

**Comment by mangotree333** [comment]
i love the dirty look!!!!

**Comment by fragtion** [comment]
I'm fine with some dirt if that is common in the real world... However, anyone else notice all the perfect concentric circular scratch-marks on the cockpit windows? Lots of scratch marks is fine, but all the perfect circles? when the lighting contrast accentuates the scratches, those circles detract from the realism for me
